styles/modules.css

```diff
@@ -0,0 +1,102 @@
+.site-header {
+  background: #4286f4;
+}
+
+.heading-group-h1 {
+  float: left;
+  padding: 0 0em .2em 1em;
+  max-width: 14em;
+}
+
+.icon-menu {
+  float: right;
+  font-size: 40px;
+}
+
+.main-nav {
+  padding: 0.1em;
+}
+
+.main-nav ul {
+  display: none;
+  padding: 0;
+  list-style: none;
+  clear: both;
+}
+
+.main-nav:hover ul {
+  display: block;
+}
+
+.main-nav li {
+  margin-right: .4em;
+  font-size: 1.5em;
+  padding: .5em .2em;
+  border-bottom: 1px solid #ccc;
+}
+
+#filters {
+  padding: 0;
+  list-style: none;
+}
+
+.template {
+  display: none;
+}
+
+.byline {
+  color: #777;
+}
+
+.byline a {
+  font-weight: bold;
+  text-decoration: none;
+}
+
+.social-icons {
+  float: right;
+}
+
+.profile-image {
+  max-width: 20%;
+  max-height: 20%;
+  float: right;
+  padding-top: 1.5em;
+}
+
+@media screen and (min-width: 800px) {
```

Comment:
Great use of the media screens, your resizing works well, and looks good. I like how your blue background color increases in height when the screen size changes.